app/models/katalyst/navigation/item.rb

```diff
@@ -10,16 +10,10 @@ class Item < ApplicationRecord
 
       attr_accessor :parent, :children, :index, :depth
 
-      TARGETS = %i[self _top _blank kpop].index_by(&:itself).freeze
-
-      enum method: TARGETS, _prefix: :target
-
       def self.permitted_params
         %i[
           title
-          url
```

Comment:
Noting we will need to add `url` to `Navigation::Card.permitted_params` in frg

Reply:
@sobakasu that's now added by including the `HasLink` concern, which permits url, target, and http_method